app/serializers/theme_serializer.rb

```diff
@@ -24,11 +24,7 @@ def filename
   end
 
   def target
-    case object.target_id
-    when 0 then "common"
-    when 1 then "desktop"
-    when 2 then "mobile"
-    end
+    Theme.targets.invert[object.target_id]&.to_s
```

Comment:
hmmm much too fancy for a serializer it has to generate this inverted hash each time it serializes. if logic is on theme it makes more sense

Theme.lookup_target(object.target_id)&.to_s